tests/TypesSpec.hs

```diff
@@ -12,9 +12,11 @@ import Pact.Types.PactValue
 
 spec :: Spec
 spec = do
-  describe "JSONPersistables" testJSONPersist
-  describe "JSONColumns "testJSONColumns
--- describe "JSONModules" testJSONModules
+  describe "JSONRoundtrips" $ do
```

Comment:
No idea why a test doing JSON persistence is called TypesSpec but I needed somewhere to test unification now, before there's syntax support, so this seemed like the right place. I put JSON at least in its own test group.